platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/cli/CleanCommand.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.cli;
+
+import com.swirlds.cli.PlatformCli;
+import com.swirlds.cli.utility.AbstractCommand;
+import com.swirlds.cli.utility.SubcommandOf;
+import com.swirlds.common.config.EventConfig;
+import com.swirlds.common.config.StateConfig;
+import com.swirlds.common.io.utility.FileUtils;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.platform.config.AddressBookConfig;
+import com.swirlds.platform.util.BootstrapUtils;
+import java.nio.file.Path;
+import java.util.List;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        name = "clean",
+        mixinStandardHelpOptions = true,
+        description = "Delete all files generated by running the platform.")
```

Comment:
Does this also clean up all merkledb files?

Reply:
they are saved in the `saved` directory as far as i know, so yes